js/table/tablemanager.ts

```diff
@@ -346,6 +350,52 @@ class TableManager {
         tableView.displayRebuyOrAddonTime();
     }
 
+    public onPlayerCardsDealed(tableId: number) {
+        const tableView = this.getTableById(tableId);
+        if (!tableView) {
+            return;
+        }
+        tableView.setRoundNotificationCaption(0);
```

Comment:
Do not use magic numbers here, either use Enum, or directly pass message which you want to set in the UI. Messages which are shown related not to TableView, but to the manager which decide to show message on TableView, thus all messages should be here, and not coded in the TableView also.